<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeatmapTab</class>
 <widget class="QMainWindow" name="HeatmapTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="orderBar" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="orderLabel">
         <property name="text">
          <string>Protein Order:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="orderSelect"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="HeatmapView" name="view">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
       </brush>
      </property>
      <property name="renderHints">
       <set>QPainter::TextAntialiasing</set>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::ScrollHandDrag</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Heatmap Tools</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionToggleSingleCol"/>
   <addaction name="actionLockOrder"/>
   <addaction name="actionSavePlot"/>
  </widget>
  <action name="actionToggleSingleCol">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="object-columns">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Single Column</string>
   </property>
   <property name="toolTip">
    <string>Don't distribute heatmap along several columns</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionSavePlot">
   <property name="icon">
    <iconset theme="camera-photo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
   <property name="toolTip">
    <string>Save the plot to SVG or PNG file</string>
   </property>
   <property name="shortcut">
    <string>Print</string>
   </property>
  </action>
  <action name="actionLockOrder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="lock">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Lock Order</string>
   </property>
   <property name="toolTip">
    <string>Do not re-order when reference changes</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HeatmapView</class>
   <extends>QGraphicsView</extends>
   <header>src/heatmap/heatmapview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
